addons/diagnostic/fnc_initTargetDebugConsole.sqf

```diff
@@ -14,6 +14,9 @@ Author:
 ---------------------------------------------------------------------------- */
 
 #define COUNT_WATCH_BOXES 8
+#define EXEC_RESULT ([nil] apply {[] call _this} param [0, text "<any>"])
```

Comment:
is nil used here to be as cheap as possible? a number would be cheaper.

Reply:
This is the debug console. It doesn't matter.

Reply:
Why the eyes? I'd rather `_x` return <any> than a random number or whatever.